coalib/parsing/filters/FilterBase.py

```diff
@@ -0,0 +1,27 @@
+from abc import ABC, abstractmethod
+
+
+def overrides(interface_class):
+    def overrider(method):
+        assert(method.__name__ in dir(interface_class))
+        return method
+    return overrider
```

Comment:
This is just to impose more restrictions to reduce errors and make the code a bit more self-documented.

We can remove this too, if not inconsistent. I've tried removing it in local and everything works fine. I can push changes if you want.